docs/2.9.0/docs/tools/daml-shell/index.rst

```diff
@@ -239,6 +265,92 @@ as long as they have the same name.
 The auto-completion provides both FQN variants (with and without package
 ID).
 
+Filtering with ``where`` clauses
+================================
+
+To refine your queries when listing contracts, you can use ``where``
+clauses to filter based on specific payload fields. This uses a SQL-like
+syntax for conditionals, and is supported for the ``active``,
+``creates``, ``archives`` and ``exercises`` commands.
+
+Nested field names are identified using dot notation such as
+``parent.child.value``.
+
+Supported comparison operators
+==============================
+
+-  ``=``: Equal to
```

Comment:
I think it's more readable without the colons after the operators, but your call.

(but keep the colons for the Logical operators, because there they aid readability) 🙃